doc/man3/RAND_DRBG_new.pod

```diff
@@ -119,12 +114,13 @@ RAND_DRBG_uninstantiate()
 clears the internal state of the B<drbg> and puts it back in the
 uninstantiated state.
 
+RAND_DRBG_evp_rand() returns the underlying EVP_RAND instance used by B<drbg>.
```

Comment:
Deprecation of RAND_DRBG and much of RAND is planning as a seperate PR.

The API is necessary to get the master EVP_RAND from the RAND_DRBG.  Since the setup needs a rethink and design (post 3.0), it might be best to just remove this API for the moment.

Reply:
I've removed this function, better it isn't public until we know it is required.